schemas/validationTest.schema.tpl.json

```diff
@@ -0,0 +1,60 @@
+{
+    "_type": "https://openminds.ebrains.eu/computation/ValidationTest",
+    "_extends": "/core/schemas/products/researchProduct.schema.tpl.json",
+    "required": [
+      "developer"
+    ],
+    "properties": {
+      "developer": {
+        "type": "array",
+        "minItems": 1,
+        "uniqueItems": true,
+        "_instruction": "Add one or several developers (person or organization) that contributed to this validation test.",
+        "_linkedCategories": [
+          "legalPerson"
+        ]
+      },
+      "digitalIdentifier": {
+        "_instruction": "Add the globally unique and persistent digital identifier of this research product. Note that this digital identifier will be used to reference all attached research product versions.",
+        "_linkedTypes": [
+          "https://openminds.ebrains.eu/core/DOI"
+        ]
+      },
+      "hasVersion": {
+        "type": "array",
+        "minItems": 1,
+        "uniqueItems": true,
+        "_instruction": "Add one or several versions of this validation test.",
+        "_linkedTypes": [
+          "https://openminds.ebrains.eu/computation/ValidationTestVersion"
+        ]
+      },
+      "scope": {
+        "_instruction": "Add the scope of this validation test.",
+        "_linkedTypes": [
+          "https://openminds.ebrains.eu/controlledTerms/ModelScope"
```

Comment:
I would have expected here a different set of terms. I would have assumed that the ModelScope is for example "single cell" and the ValidationTestScope refers to the scope of the test not the model it is testing.

Reply:
It is supposed to be the same set of terms for models and tests. The usage is, for example, "show me all validation tests that are potentially relevant for my single cell model".

Reply:
okay